static/styles/billing.scss

```diff
@@ -0,0 +1,239 @@
+            font-weight: 500;
+        }
+    }
+
+    .payment-schedule {
+        label {
+            display: inline-block;
+        }
+        input[type="radio"] {
+            display: none;
+            &:checked {
+                + .box {
+                    background: #69b190;
+                    color: white;
+                    border-color: #438667;
+                }
+            }
+        }
+        .box {
+            width: 120px;
+            height: 70px;
+            background: #efefef;
+            transition: all .2s ease;
+            display: inline-block;
+            text-align: center;
+            cursor: pointer;
+            position: relative;
+            border: 1px solid #e7e7e7;
+            border-radius: 8px;
+            margin: 0 10px 5px 0;
+            padding: 30px 20px;
+            vertical-align: top;
+            &:hover {
+                background: #e7e7e7;
+                border-color: #cbcbcb;
+            }
+            .schedule-time {
+                font-weight: bold;
+                font-size: 1.2em;
+                margin-top: 10px;
+            }
+            .schedule-amount {
+                margin-top: 5px;
+                font-size: 1.1em;
+                height: 50px;
+            }
+            .schedule-amount-2 {
+                font-size: 0.9em;
+            }
+        }
+    }
+
+    .stripe-button-el {
+        padding: 11px 25px 11px 25px;
+        font-weight: 400;
+        color: white;
+        background: #61b1b8;
+        background: linear-gradient(145deg, rgb(76, 181, 205), rgb(37, 177, 151));
+        box-shadow: 0px 3px 10px rgba(0, 0, 0, 0.2);
+        border: 0;
+        height: 40px;
+        margin: 5px 0 0 0;
+        span {
+            background: 0;
+            box-shadow: none;
+            -webkit-box-shadow: none;
+            font-family: 'Source Sans Pro', Helvetica, Arial;
+            font-size: 1.4em;
+            line-height: 20px;
+        }
+    }
+
+    .stripe-button-el:hover {
+        background: rgb(37, 177, 151);
+        box-shadow: 0px 3px 10px rgba(0, 0, 0, 0.3);
+    }
+
+    .stripe-button-el:active,
+    .stripe-button-el:not(:disabled):active {
+        background: rgb(25, 139, 118);
+        span {
+            background: 0;
+            box-shadow: none;
+            -webkit-box-shadow: none;
+        }
+    }
+
+    #error-message-box {
+        margin-top: 10px;
+        font-weight: 600;
+        display: none;
+    }
+
+    #loading-section {
+        display: none;
+    }
+
+    #loading-ellipsis {
+        width: 10%;
+        margin: 0 auto;
+    }
+
+    #loading-text {
+        margin: -5px 0px 0px 0px;
+        text-align: center;
+        font-weight: 300;
+    }
+
+    #card-updated-message {
+        text-align: center;
+        display: none;
+    }
+
+    #update-card-button-span {
+        display: block;
+        min-height: 30px;
+    }
+
+    // Style for loading indicator
```

Comment:
This looks good, but we should use the standard circle indicator we use elsewhere in the webapp, for consistency.